server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java

```diff
@@ -180,13 +180,11 @@ public static DoSDashboardRefreshResponse createResponse(final DoSDashboard dash
       switch (cta.audit()) {
         case COMPARISON:
           final Map<Integer, Integer> discrepancy = new HashMap<>();
-          int optimistic = Integer.MIN_VALUE;
-          int estimated = Integer.MIN_VALUE;
+          int optimistic = 0;
+          int estimated = 0;
           audited_contests.put(cta.contest().id(), cta.reason());
 
-          // FIXME does looking up the ComparisonAudit by name make sense?
           for (final ComparisonAudit ca : ComparisonAuditQueries.matching(cta.contest().name())) {
-            // FIXME The dashboard looks funny here.
             optimistic = Math.max(optimistic, Math.max(0, ca.optimisticSamplesToAudit() - ca.getAuditedSampleCount()));
             estimated = Math.max(estimated, Math.max(0, ca.estimatedSamplesToAudit() - ca.getAuditedSampleCount()));
```

Comment:
Can and should we reuse `ComparisonAudit.estimatedRemaining()` here?

Reply:
Oh yes, we should.